src/helpers/queries.js

```diff
@@ -5,7 +5,7 @@
  */
 export function getMeetings(date) {
   return `query getMeetings{
-  meetings(where: {date_gt: "${date.toISOString()}"} sort: "date"){
+  meetings(where: {date_gt: "${date.toISOString()}", isHidden: "false"} sort: "date"){
```

Comment:
shouldn't it be `isHidden: false`? (without " ")

Reply:
I just remembered - we need to use an operator..
So it should work like this: isHidden_ne: true
(which means "isHidden not equals true")

Reply:
updated in branch